jkamz/Questioner
# #163368826 add challenge3 folder structure

### What does this PR do?
This PR creates the directory structure for v2

### Description of the tasks to be completed
The tasks for this PR include:

- Create folders and include init files

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ch-v2folder-structure-163368826`

### What are the relevant Pivotal Tracker stories?
#163368826
